package.json

```diff
@@ -4,7 +4,11 @@
   "main": "index.js",
   "scripts": {
     "start": "NODE_ENV=production node index.js",
-    "test:ci": "echo 'no tests for unleash-docker';"
+    "test:ci": "echo 'no tests for unleash-docker';",
+    "docker:build": "./tools/release.js build",
+    "docker:build-default": "./tools/release.js build --node-docker-versions 12-alpine,14-alpine --default-node-docker-version 12-alpine",
+    "docker:publish": "./tools/release.js publish",
+    "docker:publish-default": "./tools/release.js publish --node-docker-versions 12-alpine,14-alpine --default-node-docker-version 12-alpine"
```

Comment:
here are the scripts. the CLI will yell at you if you invoke it would the proper arguments:

![image](https://user-images.githubusercontent.com/1404650/106678673-791a7980-65bb-11eb-852f-fa63bd7aed86.png)